src/Tests/Tests/epv_test_validate_php_functions.php

```diff
@@ -175,12 +176,24 @@ private function validate(PHPFileInterface $file)
 	private function checkInPhpBB(array $stmt)
 	{
 		$ok = true;
+
+		foreach ($stmt as $key => $statement)
+		{
+			if ($statement instanceof Declare_)
+			{
+				unset($stmt[$key]);
+			}
+		}
+
+		// Re-index from 0
+		$stmt = array_values($stmt);
+
 		// Lets see if there is just a namespace + class
 		if (sizeof($stmt) == 1 && $stmt[0] instanceof Namespace_)
 		{
 			foreach ($stmt[0]->stmts as $st)
 			{
-				if ($st instanceof Class_ || $st instanceof Interface_ || $st instanceof Use_)
+				if ($st instanceof Class_ || $st instanceof Interface_ || $st instanceof Use_ || $st instanceof Declare_)
```

Comment:
Please add : 
```suggestion
				if ($st instanceof Class_ || $st instanceof Interface_ || $st instanceof Use_ || $st instanceof Declare_ || $st instanceof Trait_)
```

for Fix #102